_posts/en/meetings/2018-07-19-meeting.md

```diff
@@ -0,0 +1,304 @@
+look for, called Output Script Descriptors, and he has PR [#13697][] open
+to add support for that to `scantxoutset` instead of the current ways
+keys and scripts are described to the wallet.
+
+**Discussion ([log][log scantxoutset api]):** Wuille requested and
+introduced the topic, "first of all, this is part a bigger effort to
+combine keys and scripts and [HD wallet] chains into one concept.
+There's a mini language to specify (sets of) scriptPubKeys, so I'd very
+much first want to hear comments on that language.  The other question
+is [whether] `scantxoutset` [is] experimental or not, and [is it so]
+with descriptor support in 0.17 or not?"
+
+Wladimir van der Laan and Jonas Schnelli both said they liked the idea
+of output script descriptors, and they also both supported marking
+`scantxoutset` as experimental for the 0.17 release, which will allow the
+project to freely change its API for subsequent releases and make it
+easy to incorporate feedback from users of the new RPC and new output
+script descriptors language.
+
+Luke Dashjr asked, "Should [output script descriptors] be a BIP?  Seems
+potentially useful outside Core."  Wuille replied, "potentially yes, but
+not in the first instance.  I expect that this will evolve rather
+quickly."
+
+**Conclusion:** No objections were raised to the idea of marking
+`scantxoutset` experimental for the 0.17 release, or to using output
+script descriptors if [#13697][] passes review.
+
+## Bitcoin-seeder under bitcoin-core GitHub organization
+
+**Background:** although Bitcoin uses a peer-to-peer protocol, nodes
+started for the first time don't know the IP addresses of any peers to
+connect to (except for some fallback addresses for last-ditch use), so
+they request a list of IP address for recently-active notes from a
+program called a *Bitcoin seeder.*  The node then connects to those
+peers and those peers can tell the new node about other peers, so all
+future connections can often be made completely decentralized---but a
+node can use the seeder again if decentralized peer finding isn't
+working for it.
+
+There are several seeder programs written by different authors.  One of
+them is maintained by Pieter Wuille and is simply called `bitcoin-seeder`. 
+
+**Discussion ([log][log seeder]):** Lucas Betschart requested and
+introduced the topic: "I thought because there are a few open issues and
+simple PRs for bitcoin-seeder [that] it might make sense that several
+Bitcoin maintainers have merge rights."
+
+Wuille replied, "No problem as far as I'm concerned, but I'm not sure
+it's the right message."  Luke Dashjr echoed that sentiment.
+
+**Conclusion:** Sjors Provoost suggested, "another approach could be for
+[Wuille] to give more people access to that repository?"  Wuille replied,
+"I'm fine with that!"  Betschart said, "That's also fine with me."
+
+## Moving away from Bitcoin.org
+
+**Background:** Since the 2009 release of the original Bitcoin software,
```

Comment:
I think this is ok, but some of the conjunctions are perhaps a little loaded (**But** over time..., **so** Bitcoin Core began.., etc)

I've made an attempt at rewriting the section. Feel free to take or leave:

**Background:** Since the 2009 release of the original Bitcoin software, Bitcoin (and later Bitcoin Core) resources have been hosted on Bitcoin.org. Over time, this was augmented with additional resources about the increasingly diverse Bitcoin network.

In December 2015 Bitcoin Core began using its own domain to host its resources, and later also to host its software releases. Release announcements and software continue to be mirrored to Bitcoin.org.

**Discussion:** Andrew Chow introduced the topic by suggesting "moving away from Bitcoin.org". He added "we still link to bitcoin.org for things like downloads. [We] should probably change those". See the [log][log bco] for the full discussion. Some discussion also occurred [after the meeting][post-meeting bco].

**Conclusion:** No explicit conclusion.  Post-meeting discussion seemed to indicate no significant changes to the current process.

Reply:
Used this text verbatim except for two capitalization changes to match the style used elsewhere in the document.  Thanks!